Cargo.toml

```diff
@@ -2,7 +2,6 @@
 name = "scarb-eject"
 version = "1.1.0"
 edition = "2021"
-rust-version = "1.69"
```

Comment:
this field doesn't determine what Rust version you're using, but rather it defines MSRV, which tells on which compiler versions this crate compiles

it doesn't really make sense for binary-distributed programs imho

Reply:
It was also [removed](https://github.com/software-mansion/scarb/blob/d58f2fd9d1662ff5dd1bc3c0eec8322663e4b19f/scarb-metadata/CHANGELOG.md?plain=1#L45) in Scarb since we last made changes to this repo, so I thought it's appropriate to remove it here as well